platform/bukkit/plugin/src/main/java/me/lokka30/treasury/plugin/bukkit/metrics/BukkitMetricsManager.java

```diff
@@ -0,0 +1,76 @@
+package me.lokka30.treasury.plugin.bukkit.metrics;
+
+import me.lokka30.treasury.api.economy.EconomyProvider;
+import me.lokka30.treasury.api.economy.misc.EconomyAPIVersion;
+import me.lokka30.treasury.api.economy.misc.OptionalEconomyApiFeature;
+import me.lokka30.treasury.plugin.bukkit.TreasuryBukkit;
+import org.bstats.bukkit.Metrics;
+import org.bstats.charts.SimplePie;
+import org.bukkit.Bukkit;
+import org.bukkit.plugin.RegisteredServiceProvider;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class BukkitMetricsManager {
+
+    private final TreasuryBukkit plugin;
+    public BukkitMetricsManager(final @NotNull TreasuryBukkit plugin) { this.plugin = plugin; }
+
+    Metrics metrics;
+
+    public void load() {
+        metrics = new Metrics(plugin, 12927);
+
+        metrics.addCustomChart(new SimplePie("economy-provider-name", () -> {
+            final RegisteredServiceProvider<EconomyProvider> economyProvider = getEconomyProviderRegistration();
+            return economyProvider == null ? "None" : economyProvider.getPlugin().getName();
+        }));
+
+        metrics.addCustomChart(new SimplePie("economy-provider-supports-negative-balances", () -> {
+            final RegisteredServiceProvider<EconomyProvider> economyProvider = getEconomyProviderRegistration();
+            return economyProvider == null ? null :
+                    economyProvider.getProvider().getSupportedOptionalEconomyApiFeatures()
+                            .contains(OptionalEconomyApiFeature.NEGATIVE_BALANCES) + "";
```

Comment:
Could we use `String#valueOf` instead of forcing a conversion to string appending `""`?

Reply:
Boolean.toString would also work here.

Reply:
Works too